test/units/libraries/LPMath.t.sol

```diff
@@ -14,6 +14,159 @@ contract LPMathTest is HyperdriveTest {
     using FixedPointMath for uint256;
     using HyperdriveUtils for *;
     using Lib for *;
+    using LPMath for *;
+
+    function test__calculateInitialReserves() external {
+        // NOTE: Coverage only works if I initialize the fixture in the test function
+        MockLPMath lpMath = new MockLPMath();
+
+        // Test .1% APR
+        uint256 shareReserves = 500_000_000e18;
+        uint256 initialVaultSharePrice = 0.5e18;
+        uint256 vaultSharePrice = 2.5e18;
+        uint256 apr = 0.001e18;
+        uint256 positionDuration = 365 days;
+        uint256 timeStretch = ONE.divDown(1109.3438508425959e18);
```

Comment:
I'm not in love with the magic number timestretch values used in this test.

Would be nice to have a clarifying comment around why these numbers in particular were chosen.

Ideally we'd have some formulaic generation for it, but I realize that may be infeasible or unreasonably time consuming.

Reply:
Agreed. This test was moved from `HyperdriveMath`, but I can update this.